package.json

```diff
@@ -8,8 +8,10 @@
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
-    "build": "babel ./src -d dist",
-    "start": "node dist/index.js",
+    "clean": "rm -rf ./dist && mkdir dist",
+    "build": "npm run clean && babel ./src -d dist",
+    "heroku-postbuild": "sequelize db:migrate:undo && sequelize db:migrate && sequelize db:seed:all",
+    "start": "npm run build && node dist/index.js",
```

Comment:
Don't run build with the `start` script. move the build to the `heroku-postbuild` script.

Reply:
After heroku builds (`heroku-postbuild`) you run build, you run migration:prod.

Reply:
Will do

Reply:
Changes Implemented